proposals/lip-0063.md

```diff
@@ -423,23 +423,26 @@ def createUserSubstoreArray():
             userObj.lockedBalances = getLockedBalances(account)
             userSubstore.append(userObj)
     # Append the legacy reserve account separately.
-    userSubstore.append(createLegacyReserveAccount())
+    addLegacyReserveAccount(userSubstore)
     sort userSubstore in the lexicographical order of (userObj.address + userObj.tokenID)
     return userSubstore
 
-def createLegacyReserveAccount():
-    legacyReserveAccount = account in accounts with account.address == ADDRESS_LEGACY_RESERVE
-    isEmpty = legacyReserveAmount is empty
-    legacyReserve = {}
-    legacyReserve.address = ADDRESS_LEGACY_RESERVE
-    legacyReserve.tokenID = TOKEN_ID_LSK
-    legacyReserve.availableBalance = isEmpty ? 0 : legacyReserveAccount.token.balance
+def addLegacyReserveAccount(userSubstore):
```

Comment:
Why do we do a big refactoring here where it would have been sufficient to add one line
```
 sort legacyReserve.lockedBalances in the lexicographical order of lockedBalance.module
```

Reply:
I got confused with v3 vs v4 accounts, but these changes were wrong in the end. I reverted to the previous version and added the locked module sorting. 
The `legacyReserveAccount` represents a v3 account, `legacyReserve` represents a v4 account.